src/protobuf-net.Reflection/NameNormalizer.cs

```diff
@@ -165,7 +176,13 @@ public virtual string GetName(EnumValueDescriptorProto definition)
         {
             var name = definition?.Options?.GetOptions()?.Name;
             if (!string.IsNullOrWhiteSpace(name)) return name;
-            return AutoCapitalize(definition.Name);
+
```

Comment:
this seems very odd as an "enum value only" thing; IMO this should work everywhere or nowhere; "enum values" is *very* specific...